doc/tutorials/docbuild.rst

```diff
@@ -112,18 +114,19 @@ development computer, and then link that to the upstream repo as well:
 Install the Documentation Tools
 *******************************
 
-Our documentation processing has been tested to run with Python 3.8.10
-and these other tools:
+Our documentation processing has been tested to run with Python 3.8.10 and
+later, and these other tools:
 
-* breathe                   version: 4.31.0
-* sphinx                    version: 3.5.4
-* docutils                  version: 0.16
+* breathe                   version: 4.34.0 (>=4.2.3)
+* sphinx                    version: 5.3.0  (>=3.2.1, <6.0)
+* docutils                  version: 0.16.1 (>=0.16, <=0.17.1)
 * sphinx-rtd-theme          version: 1.0.0
-* sphinx-tabs               version: 1.3.0
-* doxygen                   version: 1.8.17
+* sphinx-tabs               version: 3.4.0 (>=1.3.0, <=3.4.0)
+* doxygen                   version: 1.8.17 (Ubuntu 20.04) and 1.9.1 (Ubuntu 22.04)
 
 Depending on your Linux version, install the needed tools. You may get a
-different (newer) version of doxygen noted above that may also work.
+different (newer) version of doxygen (installed using `apt`) than shown here,
```

Comment:
Shouldn't that have been double-backtick quotes (instead of single backticks)

Reply:
Yep. That's that I get for working on md docs for a while. I'll fix in an update.